lib/cob_index/suppress_config.rb

```diff
@@ -7,16 +7,14 @@
 
 settings(&CobIndex::DefaultConfig.indexer_settings)
 
-
-to_field "status", extract_status
-
 each_record do |record, context|
-
-  if context.output_hash["status"] != [ "deleted" ]
+  status = []
+  extract_status[record, status]
```

Comment:
I don't understand this. Why is it using hash/array accessor for a indexing rule method.

Reply:
It's not. It's one of those rubyisms... Proc instance evaluation is done with `[]` vs `()`. I really hate that about ruby because it always throws me off when I see that.  And I usually avoid using `[]` as evaluator whenever I can, but in this instance we need to do it.

Reply:
woah. 🤯